docs/tutorials/vision/large_scale_classification.md

```diff
@@ -244,30 +251,3 @@ It is often straightforward to achieve a reasonable validation accuracy, but ach
 If the batch size is too big, it can exhaust GPU memory. If this happens, you’ll see the error message “cudaMalloc failed: out of memory” or something similar. There are a couple of ways to fix this:
 - Reduce the batch size.
 - Set the environment variable `MXNET_BACKWARD_DO_MIRROR` to 1. It reduces the memory consumption by trading off speed. For example, with batch size 64, inception-v3 uses 10G memory and trains 30 image/sec on a single K80 GPU. When mirroring is enabled, with 10G GPU memory consumption, we can run inception-v3 using batch size of 128. The cost is that, the speed reduces to 27 images/sec.
-
-## Appendix
-### Installing OpenCV
```

Comment:
Turns out `pip install opencv-python` is sufficient. So, removing this section and adding the pip install to requirements section.